arm-trafficmanager/2017-03-01/swagger/trafficmanager.json

```diff
@@ -0,0 +1,845 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "TrafficManagerManagementClient",
+    "version": "2017-03-01"
```

Comment:
I believe this is a namespace broad operation and currently this operation is implemented by NRP team (although I am not sure if they expose it in Swagger)